### DISCUSSION TITLE
Miners adding excess accounting data to the network

### DISCUSSION THREAD

**Discussion by didlie** [dogecoin/dogecoin#2741] [General]
A fast overview of today's blocks shows the continued abuse of the blockchain by miners adding excessive accounting information to transactions added to the blockchain, contributing to bloat.

Addresses are recorded sending multiple amounts in single transactions. This excess data is recorded on every full core and these irrelevent (irrelevant to the network and final ballances) data points are multiplying bit lengths. The end result is blockchain bloat and network storage of irrelevant information

**Comment by patricklodder** [comment]
Please stick to the original conversation.
